tests/braille-specs/de-ch-g1.yaml

```diff
@@ -1,15 +0,0 @@
-display: de-eurobrl6.dis
-table:
-  locale: de
-  grade: 1
-  __assert-match: de-g1.ctb
-tests:
```

Comment:
What happened to this test? I cannot find it in `de-g1.yaml`.

Reply:
I dropped it.

Reply:
What do you mean? Was it no longer valid?